Installing-and-Configuring/Upgrading.md

```diff
@@ -157,10 +154,10 @@ sealtotp.sh
 This needs the TPM owner password to be able to define the NVRAM space.
 (todo: [issue #151](https://github.com/osresearch/heads/issues/151)).
 
-Resealing the disk encryption keys
+Resealing the disk encryption keys (optional)
 ---
 
-When you get to the standard boot menu and after you verify the TOTP, select 'm'
+If you want to store the disk decryption key in the TPM you will need to do that now.  From the recovery shell run generic-init.  When you get to the standard boot menu and after you verify the TOTP, select 'm'
```

Comment:
Please be consistent on vocabulary.

Disk Unlock Key is defined while setting a new boot default.
Replace usb token with USB Security dongle
We need consistency in language else we get users lost

Reply:
WIP  I agree on vocabulary and have seen multiple references to this hard device by different names.  I will do a pass on the entire set of docs to standardize on 'USB Security dongle' (though I prefer security token.  one day these may not be USB devices which means we'll need to update the docs....)

I'm still not clear on the TPM disk unlock key and process.  I am familiar with LUKS keys and PGP integration.  I've never been prompted by heads to set a disk unlock while setting default boot.  This is part of the reason I don't understand this process.  (I have to reboot the system to work with heads but I'm currently using it to type this.  maybe I need to install it in a VM?  is that in a doc somewhere?)

Reply:
Purism deactivates TPM disk unlock key in board config since they pushed GPG disk unlocking through PureOS per referred above docs to their implementation.

Reply:
Heads source code uses the term 'security token' interchangeably with 'security dongle'.  I have added both terms in wiki instructions when heads uses token and replaced other references to 'security dongle'.  I also removed the use of 'smartcard' in most locations and added 'aka smartcard' as a note in a few places.